core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/documentation/SelectingColumns.kt

```diff
@@ -0,0 +1,127 @@
+package org.jetbrains.kotlinx.dataframe.documentation
+
+import org.jetbrains.kotlinx.dataframe.ColumnSelector
+import org.jetbrains.kotlinx.dataframe.ColumnsSelector
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.api.*
+import org.jetbrains.kotlinx.dataframe.columns.ColumnReference
+import org.jetbrains.kotlinx.dataframe.columns.ColumnSet
+import org.jetbrains.kotlinx.dataframe.columns.SingleColumn
+import org.jetbrains.kotlinx.dataframe.documentation.SelectingColumns.*
+import kotlin.reflect.KProperty
+
+/** {@comment
+ * In this file we provide documentation for high-level operations such as
```

Comment:
is it possible to have comments without any classes or methods or constants

Reply:
indeed no, removed it